rune/attest.go

```diff
@@ -21,7 +21,7 @@ const (
 
 var attestCommand = cli.Command{
 	Name:  "attest",
-	Usage: "attest generates a remote attestation to the corresponding enclave container",
+	Usage: "attest gets the ias report or the local report(if the spid isn't set) to the corresponding enclave container",
```

Comment:
ias report is dedicated for sgx. Using remote or local report is good description.